docs/api-docs/channels/channels-toolkit-reference.md

```diff
@@ -28,76 +34,76 @@ All channels must integrate the following endpoints.
 
 The following endpoints are recommended for most channels.
 
-| Endpoint                                                                                                    | Description                                                    |
-| ----------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------- |
-| [Channel Currency Assignments](https://developer.bigcommerce.com/api-reference/store-management/channels/channel-currency-assignments)|Manage channel specific currency settings|
-| [Price Lists](https://developer.bigcommerce.com/api-reference/store-management/price-lists)                 | Control variant-level pricing by channel, customer group, etc  |
-| [Store Information](https://developer.bigcommerce.com/api-reference/store-management/store-information-api) | Get store metadata                                             |
-| [Shipping](https://developer.bigcommerce.com/api-reference/store-management/shipping-api)                   | Manage how products are shipped                                |
-| [Webhooks](https://developer.bigcommerce.com/api-reference/webhooks)                                        | Get notified when specific events occur in a BigCommerce store |
-| [Routes](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api)                    | Create and manage page routes for headless storefronts         |
-| [Sites](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api)                     | Create and manage sites associated with a channel              |
-| [Settings](https://developer.bigcommerce.com/api-reference/store-management/settings)                       | Read global and channel specific settings                      |
+| Endpoint                                                                                                                               | Description                                                    |
```

Comment:
Please remove the extra space.

Reply:
I will undo the table formatting changes. The idea was to make them more readable in markdown format (assuming it has line-wrapping turned off) by having columns align. We should revisit this and agree on a style. They all render the same in HTML, but they're inconsistent within this page. This table uses dashes to align the columns in the markdown, whereas the next table down uses a single dash to signify a column.